### PR TITLE
UI

### PR DESCRIPTION
**Pull request by Merhallon**

### CODE REVIEW

File: tests/backend/positive/test_create_build.py [hunk 1]
```diff
@@ -1,19 +1,44 @@
...
+    @allure.link(url='', name='Documentation')
+    @allure.description("The test checks the creation of a new build check boundary value for id")
+    @pytest.mark.parametrize('characters', [1, 120, 224])
+    def test_create_build_id_validation(self, super_admin, create_project, characters):
         project_data_cre = create_project()
         build_data = BuildData.create_build_data(project_data_cre.id)
+
+        first_letter = faker_instance.random.choice(string.ascii_letters)
+        rest_characters = ''.join(faker_instance.random.choices(string.ascii_letters + string.digits, k=characters))
+        build_data.id = first_letter + rest_characters
```
Comment (Septenal14) [inline tests/backend/positive/test_create_build.py:40]: Заметил у тебя такую конструкцию в позитивных и негативных кейсах.
В виде улучшения я бы рекомендовал делать это в виде методов в utils.py, а затем использовать их здесь или забирать через фикстуру. Можно, например, сделать с передачей параметра кол-ва символов (например для негатива 256 где-то)

**Review by Septenal14 — APPROVED** [review]
Хорошая работа!! Ты прошла в первый раунд :)
Оставил рекомендацию по поводу переноса логики генерации fake данных в utils.py из теста
